.stylelintrc

```diff
@@ -0,0 +1,29 @@
+{
+    "extends": [
+        "stylelint-config-standard-scss"
+    ],
+    "plugins": [
+        "stylelint-selector-bem-pattern"
+    ],
+    "ignoreFiles": ["sass/tna-toolkit/**/*.scss"],
+    "rules": {
+        "at-rule-empty-line-before": null,
+        "block-no-empty": null,
+        "declaration-empty-line-before": null,
+        "property-no-vendor-prefix": null,
+        "value-no-vendor-prefix": null,
+        "value-keyword-case": null,
+        "scss/dollar-variable-empty-line-before": null,
+        "scss/double-slash-comment-empty-line-before": null,
+        "selector-class-pattern": null,
+        "plugin/selector-bem-pattern": {
+            "preset": "bem"
+        },
+        "declaration-block-no-duplicate-properties": {
+            "ignore": ["consecutive-duplicates-with-different-values"]
```

Comment:
This is to allow IE11 property fallbacks which we have in places

Reply:
It is throwing an error message when running: "Invalid Option: Unexpected option value "{"ignore":["consecutive-duplicates-with-different-values"]}" for rule "declaration-block-no-duplicate-properties" - I am not sure why as it works and is valid: https://stylelint.io/user-guide/rules/declaration-block-no-duplicate-properties

Reply:
I saw a few being removed when the linting rules didn't have that - here are a few examples

https://github.com/nationalarchives/ds-wagtail/blob/cfa2dd90a25764fd4d03f8749a87be8dd610bf3b/sass/tna-toolkit/bootstrap/_reboot.scss#L374-L379

https://github.com/nationalarchives/ds-wagtail/blob/cfa2dd90a25764fd4d03f8749a87be8dd610bf3b/sass/tna-toolkit/bootstrap/_tables.scss#L170-L172

https://github.com/nationalarchives/ds-wagtail/blob/cfa2dd90a25764fd4d03f8749a87be8dd610bf3b/sass/includes/search/_featured-search.scss#L10-L29

Reply:
I've removed this rule now as it was allowing more than just consecutive duplicate properties. ie11 issue resolved.